assets/src/blocks/Happypoint/HappypointBlock.js

```diff
@@ -2,11 +2,13 @@ import {HappypointEditor} from './HappypointEditor';
 import {HappypointBlock as HappypointBlockV1} from './deprecated/HappypointBlocklV1.js';
 
 const {registerBlockType, getBlockType} = wp.blocks;
+const {__} = wp.i18n;
 
 export const registerHappypointBlock = () => {
   if (!getBlockType('planet4-blocks/happypoint')) {
     registerBlockType('planet4-blocks/happypoint', {
       title: 'Happypoint',
+      description: __('The happy point block embeds (via iFrame) a “Subscribe” or engagement form on top of a full-width background image.', 'planet4-blocks-backend'),
```

Comment:
It's probably nit-picky of me, but I feel like we should stick to a way of writing this block name? Sometimes it's `Happypoint`, here `happy point`, later `Happy Point`...

Reply:
Agreed!
Let me open a new PR for this change, to keep this PR cleaner and within the ticket scope.
Just to be sure: the correct name is "Happypoint", right?

Reply:
Done https://jira.greenpeace.org/browse/PLANET-7611